### ISSUE TITLE
Training Crash (Warning: Moving average ignored a value of nan/inf && /pytorch/aten/src/ATen/native/cuda/Loss.cu:102: operator(): block: [155,0,0], thread: [33,0,0] Assertion `input_val >= zero && input_val <= one` failed.)

### ISSUE THREAD

**Issue by kprastey** [dbolya/yolact#608]
![image](https://user-images.githubusercontent.com/46772194/107907886-cf53c900-6f7a-11eb-880d-c0b6e8aa1448.png)

Not able to train on  a custom annotated dataset. The training crashes after a few epochs. Please look into this error and help resolve this... 

Environment info: 
Training on google colab with:
![image](https://user-images.githubusercontent.com/46772194/107908208-7b95af80-6f7b-11eb-86ea-8ee292b5aa62.png)

The dataset contains 1500 annotated images (1800x1600 each).

-Also let me know if you need any other information.

**Comment by code-wangshuyi** [comment]
I have also got this training crash.
Pytorch version is '1.8.0a0+1606899'.
CUDA version is 11.2

**Comment by kprastey** [comment]
@code-wangshuyi did you find out the reason for this crash?
